.ebextensions/04_create_apache_conf.config

```diff
@@ -0,0 +1,30 @@
+#
+# Create an Apache .conf file for the preview service.
+#
+files:
+  '/tmp/suitec-preview-service.httpd.conf' :
+    mode: '000755'
+    owner: root
+    group: root
+    content: |
+      # Elastic Beanstalk Managed
+      #
+      # Elastic Beanstalk managed configuration file
+      # Some configuration of httpd can be by placing files in /etc/httpd/conf.d/
+      # using Configuration Files.
+      # http://docs.amazonwebservices.com/elasticbeanstalk/latest/dg/customize-containers.html
+      #
+      # This file was overwritten during deployment
+      # by .ebextensions/05_overwrite_apache_conf.config
+
+      Listen 80
+
+      <VirtualHost *:80>
+          DocumentRoot "/var/www/html"
```

Comment:
@paulkerschen again, in the spirit of convention, the quotes in DocumentRoot path seem unnecessary